### PR TITLE
Lock control

### PR DESCRIPTION
**Pull request by travisleo**

### CODE REVIEW

File: ece595_project_2/source/server.h [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#ifndef SERVER_H_
+#define SERVER_H_
+
+#include "mdns.h"
+
+extern void Run_HTTPServer(void);
+extern void server_thread(void *arg);
```
Comment (djaenicke) [inline ece595_project_2/source/server.h]: Is this needed?

File: ece595_project_2/source/main.c [hunk 2]
```diff
@@ -47,37 +64,42 @@ typedef struct Task_Cfg_Tag
...
-
     SYSMPU_Type *base = SYSMPU;
     BOARD_InitPins();
     BOARD_InitBootPeripherals();
     BOARD_BootClockRUN();
     BOARD_InitDebugConsole();
+    BOARD_Enable_SW_Interrupts();
 
-    Init_Xbee_Interface();
+    //Init_Xbee_Interface();
```
Comment (djaenicke) [inline ece595_project_2/source/main.c:102]: Uncomment this

File: ece595_project_2/source/server.h (hunk 1 above)
Comment (travisleo) [reply ece595_project_2/source/server.h]: some of it, but I fixed it

File: ece595_project_2/source/main.c (hunk 2 above)
Comment (travisleo) [reply ece595_project_2/source/main.c:102]: done

**Review by djaenicke — APPROVED** [review]

**Pull request merged**